Source/WTF/wtf/PrintStream.h

```diff
@@ -45,6 +45,11 @@ inline const char* boolForPrinting(bool value)
     return value ? "true" : "false";
 }
 
+inline const char* boolForPrinting(const std::optional<bool>&value)
```

Comment:
I'm sorry to inform you that you have to build 1800 source files again, but you need to add a space between & and value

Reply:
<img src="https://imgs.xkcd.com/comics/compiling.png" />